content/en/api/dashboard_lists/code_snippets/result.api-dashboard-list-get-items.py

```diff
@@ -14,10 +15,11 @@
             'is_favorite': True,
             'is_read_only': True,
             'type': 'integration_screenboard',
-            'id': 66,
+            'id': 'rys-xwq-geh',
```

Comment:
Same comment as the one above regarding the `id`, but another check is to ensure we use the same `id` value for this field as the one used in `url`.